code/script.js

```diff
@@ -30,18 +74,130 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
-const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+// Interaction 7, win-win.
+// As bot, hand over food item to user and wish them a happy meal (what's the English phrase?)
+const done = () => {
```

Comment:
Naming suggestion: `wishHappyMeal` or `showLastMessage` or `sayBuonAppetito` (I don't know the English phrase either 🤓 )